templates/includes/install-prepare-adb.hbs

```diff
@@ -11,7 +11,7 @@
               <input type="button" class="btn btn-primary clipboard-button" value="&#10697;"></input>
             </div>
           </div>
-          <h4>Fedora / OpenSUSE / CentOS</h4>
+          <h4>Fedora, CentOS, RHEL, etc.</h4>
```

Comment:
Agreed about adding RHEL.
For heading context, i'd like to avoid `,` and `.`
The `/` better indicates a difference to body text imo.